src/share/classes/com/sun/javatest/regtest/exec/RegressionScript.java

```diff
@@ -1342,5 +1351,14 @@ private static String getVersion() {
     private Modules modules;
     private ScratchDirectory scratchDirectory;
     Locations locations;
+    private final TestProperties properties;
+
+    private RegressionScript() {
+        this(null);
```

Comment:
Does this constructor get used anywhere after this change? If yes, then we might have to add a null check at the call site where we are using the `properties` field to find the default timeout configuration value, like here https://github.com/openjdk/jtreg/pull/253/files#diff-19a61ca7f0fe24f891bee1b6b95f4ab2f740636458c9b20acff20c052aed9223R539